test/functional/rpc_invalid_address_message.py

```diff
@@ -109,6 +111,7 @@ def test_getaddressinfo(self):
         assert_raises_rpc_error(-5, "Invalid or unsupported Segwit (Bech32) or Base58 encoding.", node.getaddressinfo, BECH32_INVALID_PREFIX)
         assert_raises_rpc_error(-5, "Invalid or unsupported Base58-encoded address.", node.getaddressinfo, BASE58_INVALID_PREFIX)
         assert_raises_rpc_error(-5, "Invalid or unsupported Segwit (Bech32) or Base58 encoding.", node.getaddressinfo, INVALID_ADDRESS)
+        "isscript" not in node.getaddressinfo(BECH32_VALID_UNKNOWN_WITNESS)
```

Comment:
This is a no-op, I think you meant to assert?
```suggestion
        assert "isscript" not in node.getaddressinfo(BECH32_VALID_UNKNOWN_WITNESS)
```